components/Summary/Summary.module.css

```
@@ -0,0 +1,8 @@
+@media (min-width: 1536px) {
```

```suggestion
@media (min-width: 1536px) {
```

I would suggest documenting all of the breakpoints that you use in a global styles file using comments (if those cannot be put into a variable.